draft-ietf-quic-http.md

```diff
@@ -1111,12 +1110,14 @@ window of these streams will increase the chance that the remote peer reaches
 the limit early and becomes blocked. In particular, implementations should
 consider that remote peers may wish to exercise reserved stream behavior
 ({{stream-grease}}) with some of the unidirectional streams they are permitted
-to use. To avoid blocking, the transport parameters sent by both clients and
-servers MUST allow the peer to create at least one unidirectional stream for the
-HTTP control stream plus the number of unidirectional streams required by
-mandatory extensions (three being the minimum number required for the base
-HTTP/3 protocol and QPACK), and SHOULD provide at least 1,024 bytes of flow
-control credit to each stream.
+to use.
+
+Each endpoint needs to create at least one unidirectional stream for the HTTP
+control stream. QPACK requires two additional unidirectional streams, and other
+extensions might require further streams. Therefore, the transport parameters
+sent by both clients and servers MUST allow the peer to create at least three
+unidirectional streams. These transport parameters SHOULD also provide at least
+1,024 bytes of flow control credit to each unidirectional stream.
```

Comment:
RFC Editor asked:

> Which of these interpretations is correct?  Also, note that this sentence is quite long and may benefit from further rephrasing to break it up.
> 
> Original:
>> To avoid blocking, the transport parameters sent by both clients and 
>> servers MUST allow the peer to create at least one unidirectional 
>> stream for the HTTP control stream plus the number of unidirectional 
>> streams required by mandatory extensions (three being the minimum 
>> number required for the base HTTP/3 protocol and QPACK), and SHOULD 
>> provide at least 1,024 bytes of flow control credit to each stream.
> 
> Perhaps A:
>> To avoid blocking, the transport parameters sent by both clients and 
>> servers 1) MUST allow the peer to create at least one unidirectional 
>> stream for the HTTP control stream plus the number of unidirectional 
>> streams required by mandatory extensions (three being the minimum 
>> number required for the base HTTP/3 protocol and
>> QPACK) and 2) SHOULD provide at least 1,024 bytes of flow-control 
>> credit to each stream.
> 
> Perhaps B:
>> To avoid blocking, the transport parameters sent by both clients and 
>> servers MUST allow the peer to create at least one unidirectional 
>> stream for the HTTP control stream plus the number of unidirectional 
>> streams required by mandatory extensions (three being the minimum 
>> number required for the base HTTP/3 protocol and QPACK), and the peer 
>> SHOULD provide at least 1,024 bytes of flow-control credit to each 
>> stream.